app/javascript/components/Articles/ArticlesList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import ArticlesDetails from './ArticlesDetails';
+import { array } from "prop-types";
+
+export interface ArticlesListProps {
+    link?:string
+}
+
+export default function ArticlesList(props) {
+
+    let display = (<h3>It looks as though there's nothing here yet!</h3>)
+
+    function formatDate(date) {
+        let msec = Date.parse(date)
+        let newDate = new Date(msec)
+        return newDate.toDateString()
+    }
+
+    function getArticleLink(id) {
+        let baseURL = "http://localhost:3001/articles/"
```

Comment:
You don't need to give the whole url in a link, as if you give a relative url it will stay on the same domain. So something like `` return `/articles/${id}` ``.